bot/commands/misc/roll.js

```diff
@@ -26,6 +26,6 @@ module.exports = new Command('roll', (msg, args) => {
 
 // TODO: add args
 module.exports.help = {
-	args: '',
+	args: '<lowest number> <highest number>',
```

Comment:
Yuuko's built-in commands use square brackets to denote optional arguments. Let's use that here too for consistency.

```suggestion
	args: '[lowest number] [highest number]',
```